src/Sylius/Bundle/ReviewBundle/Resources/public/js/sylius-review.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$(document).ready(function(){
+    $("#reviewForm").submit(function(e){
+        e.preventDefault();
+        var form = $(this);
+
+        $.ajax({
+            type: "POST",
+            url: location.href+"/product_review",
+            data: $(this).serialize(),
+            dataType: "json",
+            success: function(data) {
+                if (data == 'success') {
```

Comment:
We should check the status code of the response:

201 - Created
400 - Bad request (validation failed, most likely)
500 - Internal server error

We are really sending 'success' string in the controller?